src/views/network/IpAddressesTab.vue

```diff
@@ -181,6 +214,21 @@ export default {
         this.fetchLoading = false
       })
     },
+    fetchListPublicIpAddress () {
+      return new Promise((resolve, reject) => {
+        const params = {
+          zoneid: this.resource.zoneid,
+          domainid: this.resource.domainid,
+          account: this.resource.account,
+          forvirtualnetwork: true,
+          allocatedonly: false
+        }
+        api('listPublicIpAddresses', params).then(json => {
+          const listPublicIps = json.listpublicipaddressesresponse.publicipaddress || []
+          resolve(listPublicIps)
```

Comment:
Can you sort the IPs @utchoang (and make the list searching, like other a-selects)?

Reply:
@rhtyd I've fixed it.